docs/self-managed/platform-deployment/helm-kubernetes/guides/ingress-setup.md

```diff
@@ -185,6 +185,29 @@ Once deployed, you can access the Camunda Platform 8 components on:
 - **Keycloak authentication:** `https://keycloak.camunda.example.com`
 - **Zeebe Gateway:** `grpc://zeebe.camunda.example.com`
 
+## Ingress Controllers
+
+Ingress resources require the cluster to have an [Ingress Controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/) running. There are many options for configuring your Ingress Controller. If you are using a cloud provider such as AWS or GCP, we recommend you follow their Ingress setup guides if an Ingress Controller is not already pre-installed.
```

Comment:
May want to provide links to such guides for AWS and GCP.

Reply:
I wasn't able to find official guides. There is discussion of ingress controllers in the middle of their load balancing documentation, and some outside blog posts, but nothing I felt was concrete enough to link to from our official documentation. Curious if @jessesimpson36 has any resources here? Could be I just got lost in their documentation and missed it!

Reply:
@jessesimpson36 Let me know if you have preferences here -- can always adjust in a subsequent PR.